roles/observe_chargeback/tasks/chargeback_tests.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Enable Cloudkitty Module (hashmap)
+  ansible.builtin.command:
+    cmd: "{{ openstack_cmd }} rating module enable hashmap"
+  register: enable_hashmap
+  # The task is only 'changed' if the output confirms the module was enabled/changed
+  # This is the critical line to resolve the 'no-changed-when' error
+  changed_when: "'Enabled' in enable_hashmap.stdout or 'change' in enable_hashmap.stdout"
```

Comment:
If the hashmap module was already enabled, then seeing "Enabled" in the output does not indicate a changed status, since the state before and after were the same.